packages/expo-cli/src/credentials/views/IosDistCert.ts

```diff
@@ -36,10 +39,23 @@ export class CreateIosDist implements IView {
     return null;
   }
 
+  // TODO(quin): support the option to go 'back'
+  async select(ctx: Context, options: DistCertOptions): Promise<ISelect<DistCert> | DistCert> {
+    return ctx.user ? await this.create(ctx) : await this.provideOrGenerate(ctx);
```

Comment:
select in UseExistingDistributionCert is updating db and her it's not? I'm not sure what would be the desired behavior here, but it should be the same.
Are you want to update all credentials at the end or everything separately?

Reply:
It should be ok to update the credentials separately. 

The situation we wanted to avoid is generating a cert, running into an unrecoverable error and requiring the user to go to Apple Development Portal to manually revoke credentials. As long as we give people the option to recover from errors (ie) revoke certs when the max has been generated, and we let ppl use previously generated credentials (even from a past failed attempt to build), this should be an acceptable user experience.